osl/sandbox_solaris.go

```diff
@@ -0,0 +1,24 @@
+package osl
+
+// NewSandbox provides a new sandbox instance created in an os specific way
+// provided a key which uniquely identifies the sandbox
+func NewSandbox(key string, osCreate, isRestore bool) (Sandbox, error) {
```

Comment:
@mrjana could you pls chime in on the implications of not implementing the NewSandbox API ?
I know some other platforms also take this route. But am curious here because I dont have a way currently to verify this change.